angular.json

```diff
@@ -35,8 +35,13 @@
                         "scripts": [
                             "node_modules/marked/lib/marked.js"
                         ],
-                        "es5BrowserSupport": true,
-                        "showCircularDependencies": false
+                        "showCircularDependencies": false,
```

Comment:
This property `es5BrowserSupport` is deprecated?

Reply:
`es5BrowserSupport` flag has been removed from schema.